ecosystem/sep-0010.md

```diff
@@ -43,6 +47,7 @@ The authentication flow is as follows:
    1. Source account set to the **Server Account**.
    1. Value set to the **Server**'s domain that the client requested the challenge from.
 1. The **Client** verifies that if the transaction has other operations they are Manage Data operations that all have their source accounts set to the **Server Account**
+1. If the transaction has a Manage Data operation with key `client_domain`, the **Client** obtains a signature from the **Client Domain Account** and adds it to the challenge transaction
```

Comment:
❗ The additional `if` if worthwhile here to distinguish the client should only do this if it requested client verification, and not otherwise.
```suggestion
1. If the client included a client domain in the request, and the transaction has a Manage Data operation with key `client_domain`, the **Client** obtains a signature from the **Client Domain Account** and adds it to the challenge transaction
```